backend/src/main/java/com/javahelp/backend/domain/user/register/IUserRegisterInputBoundary.java

```diff
@@ -0,0 +1,22 @@
+package com.javahelp.backend.domain.user.register;
+
+/**
+ * An input boundary on how to get the info from the (given) request model for a user
+ */
+public interface IUserRegisterInputBoundary {
+
+    /**
+     * @return the input email address from the given User RequestModel passed by the View Model
+     */
+    String getEmailAddress();
+
+    /**
+     * @return the input username from the UserRequestModel
+     */
+    String getUsername();
+
+    /**
+     * @return the Base64-generated string salt combined with hash.
+     */
+    String getSaltandHash();
```

Comment:
Remember to use camelCase, `getSaltandHash` is equivalent to 'get saltand hash' in english